pkg/clusteragent/orchestrator/status.go

```diff
@@ -90,8 +90,8 @@ func GetStatus(ctx context.Context, apiCl kubernetes.Interface) map[string]inter
 	for _, node := range orchestrator.NodeTypes() {
 		if value, found := orchestrator.KubernetesResourceCache.Get(orchestrator.BuildStatsKey(node)); found {
```

Comment:
Note: This means i.e. for CLC we only show cache stats for values which exist in the `status` output